.github/workflows/test-executor-linux-arm64.yaml

```
@@ -9,7 +9,7 @@ on:
       - "*"
 
 jobs:
-  build:
+  "Local-Exec ARM64 Test":
```

nit: this will show up in the GitHub UI like `Test linux/arm64 executor / Local-Exec ARM64 Test` - the arm64 part is redundant and the local-exec part is not that important IMO since the BES UI will show "remote execution off" in the header

```suggestion
  "test":
```